packages/file_selector/file_selector_platform_interface/test/file_selector_platform_interface_test.dart

```diff
@@ -19,6 +19,16 @@ void main() {
       FileSelectorPlatform.instance = ExtendsFileSelectorPlatform();
     });
   });
+
+  group('#GetDirectoryPaths', () {
+    test('Should throw unimplemented exception', () async {
+      final FileSelectorPlatform sut = ExtendsFileSelectorPlatform();
```

Comment:
Consider renaming to `fileSelector` or similar.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations